src/lib/api/user/server/userApiClient.server.test.ts

```diff
@@ -1,5 +1,7 @@
 import { request } from '@/lib/api/common/server';
-import { fetchUserList } from '@/lib/api/user/server';
+import { fetchUserList, getUserById } from '@/lib/api/user/server';
+
+import { IGetUserByIdResponse, ITemporaryGetUserByIdResponse } from './userApiClient.server';
```

Comment:
```ts
import { fetchUserList, getUserById } from '@/lib/api/user/server';

import { IGetUserByIdResponse, ITemporaryGetUserByIdResponse } from './userApiClient.server';
```

These two lines should be imported from the same path. I suppose the interfaces are not listed in the index file.